### PR TITLE
Make /feature, /function and /config commands in their own right

### PR DESCRIPTION
**Pull request by magicus**
There is no good reason to shoehorn these into `/wynntils`. For one, they are long enough as it is, and without the `/wynntils` prefix, they are a bit shorter. Secondly, now `/wynntils` is more concentrated about Wynntilsy things, like discord links etc.

I was initially worried about clashes with Wyncraft commands, but my two previous PRs (hide admin commands, and make real commands tab expansion aware) have alleviated any such fears.

### CODE REVIEW

File: common/src/main/java/com/wynntils/commands/ConfigCommand.java [hunk 1]
```diff
@@ -2,14 +2,18 @@
  * Copyright © Wynntils 2022.
  * This file is released under AGPLv3. See LICENSE for full license details.
  */
-package com.wynntils.commands.wynntils;
+package com.wynntils.commands;
+
+import static net.minecraft.commands.Commands.literal;
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/commands/ConfigCommand.java]: Ew. Do we like static imports?

Comment (magicus) [reply common/src/main/java/com/wynntils/commands/ConfigCommand.java]: In general, no, but I thought it made the complex Brigadier node building much more readable. But if you really dislike it, I can remove them.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/commands/ConfigCommand.java]: Isn't it nice to remove it to keep to our standards?

Comment (magicus) [reply common/src/main/java/com/wynntils/commands/ConfigCommand.java]: OK, I'll fix.

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**